internal/api/v1/router.go

```diff
@@ -43,6 +44,8 @@ func jsonErrorResponse(w http.ResponseWriter, responseErrors APIErrors) {
 func errorHandler(action APIActionFunc) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
 		if errors := action(w, r); errors != nil {
+			log := tracelog.Logger(r.Context())
+			log.V(1).Info("responding with json error response", "action", action, "errors", errors)
```

Comment:
Merge the two lines, drop the temp variable ?

Reply:
Will do in another PR